<launch>

  <!-- Public ROS param for service name so other nodes can access it -->
  <param name="projector_service_name" type="string" value="command_projector" />

  <param name="projector_yaml_directory" type="string" value= "$(find sl_sensor_projector)/projector_yaml/lightcrafter4500.yaml" /> 
  <node pkg="sl_sensor_projector" type="lightcrafter_4500_node" name="lightcrafter_4500_node" output="screen"/>

</launch>
